test/test_generated_site.rb

```diff
@@ -52,9 +52,11 @@ class TestGeneratedSite < JekyllUnitTest
     end
 
     should "include a post with a abbreviated dates" do
-      refute_nil @site.posts.index { |post|
-        post.relative_path == "_posts/2017-2-5-i-dont-like-zeroes.md"
-      }
+      refute_nil(
+        @site.posts.index do |post|
+          post.relative_path == "_posts/2017-2-5-i-dont-like-zeroes.md"
+        end
```

Comment:
Maybe in all these cases what we should do is pull out the method with the block, or use a unified helper (i thought we had one?):

```ruby
post_of_interest = @site.posts.find {|post| post.relative_path == "_posts/2017-2-5-i-dont-like-zeroes.md" }
refute_nil post_of_interest
```

or, with a helper method

```ruby
refute_nil find_by(:relative_path, "_posts/2017-2-5-i-dont-like-zeroes.md", @site.posts)
```